src/DotNetWorker.Core/FunctionMetadata/DefaultFunctionMetadata.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Azure.Functions.Worker.Core.FunctionMetadata
+{
+    /// <summary>
+    /// Local representation of FunctionMetadata
+    /// </summary>
+    public class DefaultFunctionMetadata : IFunctionMetadata
```

Comment:
Is this an unused type?

Reply:
https://github.com/Azure/azure-functions-dotnet-worker/pull/956/files/2556480f2ae5797cef4fc0d2189d5c475f5ad3e1#diff-929c7a5de0578a08eef02c93993115159659489d6263f68a40aa685ba0c6124dR103 - used in the generator but as a string so it makes it hard to find on VS